storage_manager_new.py

```diff
@@ -5,184 +5,260 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.clear_self_items_list_and_load_items_from_file()
 
-    def load_items_from_file(self):
-        Item.instances = []
+    def clear_self_items_list_and_load_items_from_file(self):
```

Comment:
Don't include `self` in the name of the method. This is just language implementation detail, I'd say it is `clear_storage_and_(...)`.